src/MetricExporter.ts

```diff
@@ -306,10 +235,73 @@ export class DynatraceMetricExporter implements MetricExporter {
 		return Promise.resolve();
 	}
 
-	private _warnNormalizationFailure(name: string) {
-		if (this._failedNormalizations === 0) {
-			diag.warn(`DynatraceMetricExporter: Failed to normalize ${name}. Skipping exporting this metric.`);
+	getPreferredAggregationTemporality(): AggregationTemporality {
+		return AggregationTemporality.DELTA;
+	}
+
+	private serializeCounter(metric: MetricData): string[] {
+		const out: string[] = [];
+		if (metric.dataPointType !== DataPointType.SINGULAR) {
+			return out;
 		}
-		this._failedNormalizations = ++this._failedNormalizations % 1000;
+
+		for (const point of metric.dataPoints) {
+			const counter = this._dtMetricFactory.createCounterDelta(metric.descriptor.name, dimensionsFromPoint(point), point.value);
+			if (counter) {
+				out.push(counter.serialize());
+			}
+		}
+
+		return out;
 	}
+
+	private serializeHistogram(metric: MetricData): string[] {
+		const out: string[] = [];
+		if (metric.dataPointType !== DataPointType.HISTOGRAM) {
+			return out;
+		}
+
+		for (const point of metric.dataPoints) {
+			if (point.value.count === 0) {
+				continue;
+			}
+
```

Comment:
This is almost the same as the first 3 lines in `estimateHistogram`, right? Should we remove one of them to be less redundant?